config/api/src/main/java/org/jboss/windup/config/GraphRewrite.java

```diff
@@ -34,12 +39,34 @@ public GraphRewrite(Iterable<RuleLifecycleListener> listeners, GraphContext cont
         this.graphContext = context;
     }
 
+    /**
+     * @return The exception which holds information where Windup has stopped before finishing (typically on an external request),
+     *      or null if Windup finished normally.
+     */
+    public WindupStopException getWindupStopException()
+    {
+        return windupStopException;
+    }
+
+    /**
+     * Stores the exception which holds information if, and where, the Windup stopped (typically on an external request).
+     */
+    public void setWindupStopException(WindupStopException windupStopException)
+    {
+        if (this.windupStopException != null)
```

Comment:
Because second call to this method would mean there's a broken logic somewhere in the stop handling code.